Makefile

```diff
@@ -313,6 +311,8 @@ build-manifest: kustomize
 CLUSTER_PROVIDER ?= kind
 LOCAL_DEV_CLUSTER_VERSION ?= main
 KIND_WORKER_NODES ?=2
+BUILD_CONTAINERIZED ?= build_image
+DEV_OPTS ?= "CI_DEPLOY PROMETHEUS_DEPLOY"
```

Comment:
AFAIU `CI_DEPLOY` is not required for local development. https://github.com/sustainable-computing-io/kepler/tree/main/manifests/k8s